setup.py

```diff
@@ -62,8 +45,8 @@
     keywords="pyfesom2",
     name="pyfesom2",
     packages=["pyfesom2"],
-    #package_dir={"pyfesom2": "pyfesom2"},
-    #package_data={"": ["*.geojson"]},
+    # package_dir={"pyfesom2": "pyfesom2"},
```

Comment:
Sorry about this one, my vim seems to automatically run black formatting whenever I save a Python file... not sure when/how I put that in there ;-)

Reply:
Happens to me all the time :)